go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/nokia/danm
+
+go 1.12
+
+require (
+	github.com/apparentlymart/go-cidr v1.0.1
+	github.com/containernetworking/cni v0.7.1
+	github.com/containernetworking/plugins v0.8.5
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/intel/multus-cni v0.0.0-20191029173424-8c76fd62e0e9
+	github.com/intel/sriov-cni v2.1.0+incompatible
+	github.com/j-keck/arping v1.0.0
+	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
+	github.com/vishvananda/netlink v1.1.1-0.20200221165523-c79a4b7b4066
+	k8s.io/api v0.17.3
+	k8s.io/apimachinery v0.17.3
+	k8s.io/client-go v0.17.3
+	k8s.io/code-generator v0.17.3
+	k8s.io/kubernetes v1.14.10
```

Comment:
anything stopping us from upgrading this one too?

Reply:
I wil give it a try

Reply:
It will throw error with the natural resolution mechanism.
Related issue: [#79384](https://github.com/kubernetes/kubernetes/issues/79384)
However with the following ['gun to its head'](https://github.com/kubernetes/kubernetes/issues/79384#issuecomment-586909612) method it works.
The go.god file would look like this:
```
module github.com/nokia/danm

go 1.13

require (
        github.com/apparentlymart/go-cidr v1.0.1
        github.com/containernetworking/cni v0.7.1
        github.com/containernetworking/plugins v0.8.5
        github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
        github.com/intel/multus-cni v0.0.0-20200316130803-079c853eba60
        github.com/intel/sriov-cni v2.1.0+incompatible
        github.com/j-keck/arping v1.0.0
        github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
        github.com/vishvananda/netlink v1.1.1-0.20200221165523-c79a4b7b4066
        k8s.io/api v0.17.4
        k8s.io/apimachinery v0.17.4
        k8s.io/client-go v0.17.4
        k8s.io/code-generator v0.17.4
        k8s.io/kubernetes v1.14.10
)

replace (
        k8s.io/api => k8s.io/api v0.17.4
        k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.4
        k8s.io/apimachinery => k8s.io/apimachinery v0.17.4
        k8s.io/apiserver => k8s.io/apiserver v0.17.4
        k8s.io/cli-runtime => k8s.io/cli-runtime v0.17.4
        k8s.io/client-go => k8s.io/client-go v0.17.4
        k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.4
        k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.17.4
        k8s.io/code-generator => k8s.io/code-generator v0.17.4
        k8s.io/component-base => k8s.io/component-base v0.17.4
        k8s.io/cri-api => k8s.io/cri-api v0.17.4
        k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.17.4
        k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.17.4
        k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.17.4
        k8s.io/kube-proxy => k8s.io/kube-proxy v0.17.4
        k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.17.4
        k8s.io/kubectl => k8s.io/kubectl v0.17.4
        k8s.io/kubelet => k8s.io/kubelet v0.17.4
        k8s.io/kubernetes => k8s.io/kubernetes v1.17.4
        k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.17.4
        k8s.io/metrics => k8s.io/metrics v0.17.4
        k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.4
)
```
I had to force require them all 🤣 
code generates
all binares build
all unit tests pass

@Levovar do you approve it?

Reply:
yes please, thanks !
that sounds like a Nokia problem to me :P

Reply:
Done